build/ci/github_actions/operator/deploy_driver.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -xel
+set +o pipefail
+
+operator_image_for_test=$operator_image_repository_for_test:$operator_image_tag_for_test
+
+kind_node_name=`docker ps --format "{{.Names}}"`
+docker exec -i $kind_node_name apt-get update
+docker exec -i $kind_node_name apt -y install open-iscsi
+
+edit_cr_images (){
+  cd $(dirname $cr_file)
+  chmod 547 $(basename $cr_file)
+  declare -A cr_image_fields=(
+      [".spec.controller.repository"]="$controller_repository_for_test"
+      [".spec.controller.tag"]="$driver_images_tag"
+      [".spec.node.repository"]="$node_repository_for_test"
+      [".spec.node.tag"]="$driver_images_tag"
+  )
+  for image_field in ${!cr_image_fields[@]}; do
+      cr_image_value=${cr_image_fields[${image_field}]}
+      yq eval "${image_field} |= env(cr_image_value)" $(basename $cr_file) -i
+  done
+  cd -
+}
+
+edit_cr_images
+
+edit_operator_yaml_image (){
+  cd $(dirname $operator_yaml)
+  operator_image_in_branch=`yq eval '(.spec.template.spec.containers[0].image | select(. == "*ibmcom*"))' $(basename $operator_yaml)`
+  sed -i "s+$operator_image_in_branch+$operator_image_for_test+g" $(basename $operator_yaml)
```

Comment:
why not with `yq`?

Reply:
I tried to do it with `yq`, but it wasn't working well because that this file has multiple YAML objects in it and it is complicated to edit only the deployment there and not harm the other YAML objects

Reply:
the author [wrote that this is supported](https://github.com/mikefarah/yq/issues/9#issuecomment-398573801).
if it's too hard/`yq` has a bug, then please open a ticket for it and write its number in a comment above

Reply:
could you write this number in a code comment above the `sed` line?

Reply:
btw have you tried to use [`select`](https://stackoverflow.com/a/66451516)?

Reply:
could you add more details in the comment around the ticket number?
similar to [this](https://github.com/IBM/ibm-block-csi-driver/blob/87a76353c62edea747beb79212963645469bd94d/scripts/ci/jenkins_pipeline_csi#L52)

Reply:
done